ci-operator/templates/cluster-launch-e2e-azure.yaml

```diff
@@ -10,6 +10,8 @@ parameters:
   required: true
 - name: LOCAL_IMAGE_BIN
   required: true
+- name: IMAGE_SYNC
```

Comment:
Does the ci-operator populate this automagically?

Reply:
yes. `Local_IMAGE_{component}` takes images from staging, `Image_{componets}` takes from stable IS